FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -173,10 +251,13 @@ - (void)appDidBecomeActiveNotification:(NSNotification *)notification {
       // happens a lot later.
       // Dropping the app start trace in such situations where the launch time is taking more than
       // 60 minutes. This is an approximation, but a more agreeable timelimit for app start.
-      if (currentTimeSinceEpoch - startTimeSinceEpoch < gAppStartMaxValidDuration) {
+      if ((currentTimeSinceEpoch - startTimeSinceEpoch < gAppStartMaxValidDuration) &&
+          ![self isApplicationPreWarmed]) {
         [self.appStartTrace stop];
       } else {
         [self.appStartTrace cancel];
+        [self.activeTrace incrementMetric:kFPRPrewarmedAppStartDroppedWithDuration
```

Comment:
I don't think it is a good idea to mark the foreground/background session trace with app start duration. We should find another mechanism to record this if required.

Reply:
Will move this discussion to another PR.

Reply:
Discussed and decided that we won't be capturing duration of dropped `_as`